### ISSUE TITLE
Bybit Perpetual Testnet: trading pair symbol cannot be found

### ISSUE THREAD

**Issue by RC-13** [hummingbot/hummingbot#4372]
**Describe the bug**
When creating a perpetual market making strategy using `Bybit perpetual testnet`, there is no autocomplete list. After entering the market and starting the strategy, an `unhandled error while status polling...` will occur causing the market state to be `not ready`

```
2021-10-05 09:44:00,151 - 4778 - hummingbot.connector.derivative.bybit_perpetual.bybit_perpetual_derivative - NETWORK - Unexpected error while fetching account updates.
Traceback (most recent call last):
  File "/Users/clint/Github/kraken-tier/hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_derivative.py", line 836, in _status_polling_loop
    await safe_gather(
  File "/Users/clint/Github/kraken-tier/hummingbot/core/utils/async_utils.py", line 22, in safe_gather
    return await asyncio.gather(*args, **kwargs)
  File "/Users/clint/Github/kraken-tier/hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_derivative.py", line 793, in _update_trade_history
    "symbol": await self._trading_pair_symbol(trading_pair),
  File "/Users/clint/Github/kraken-tier/hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_derivative.py", line 266, in _trading_pair_symbol
    return await self._order_book_tracker.trading_pair_symbol(trading_pair)
  File "/Users/clint/Github/kraken-tier/hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_order_book_tracker.py", line 42, in trading_pair_symbol
    raise ValueError(f"The symbol representing trading pair {trading_pair} could not be found")
ValueError: The symbol representing trading pair BTC-USDT could not be found
```

**Steps To Reproduce**

Here are the steps to reproduce the issue (see attachments in the section below):

1. Create a perpetual market making strategy using Bybit perpetual testnet
1. When asked to enter a market, press tab or space to check if auto complete list will be displayed.
1. Enter the market where you want to trade, for my case I use BTC-USDT
1. Start the strategy
...

**Screenshots**
![image](https://user-images.githubusercontent.com/57189990/135948796-39ec8517-6415-4892-8858-1f15c2ace4dc.png)
**Release version**
0.44.0

**Attachments**

[logs_conf_perpetual_market_making_1.log](https://github.com/CoinAlpha/hummingbot/files/7282280/logs_conf_perpetual_market_making_1.log)

_WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!_

**Optional: your discord username:**

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1ran8vb) by [Unito](https://www.unito.io)

**Comment by JohnnyVicious** [comment]
This issue for Bybit perpetual testnet, happens with any market you try to configure.
I tried BTC-USD and BTC-USDT, even BTCUSD without the - but that throws another error.

**Comment by rapcmia** [comment]
Market now shows autocomplete list and fixed applied on Hummingbot version 1.4
